contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -649,10 +630,10 @@ handle_where_clause_attnums(ParseState *pstate, Node *w_clause, List *target_att
 					if (attrno == InvalidAttrNumber)
 					{
 						ereport(ERROR,
-						(errcode(ERRCODE_UNDEFINED_COLUMN),
-						 errmsg("column \"%s\" of relation \"%s\" does not exist",
-								name,
-								RelationGetRelationName(pstate->p_target_relation))));
+								(errcode(ERRCODE_UNDEFINED_COLUMN),
```

Comment:
nit: unnecessary diff.

Also, is this case different than the others? ie throwing an error rather than returning null result set?

Reply:
Updated the unnecessary change in latest rev
This is a different error and expected. This will be encountered when user tries to perform sp_descirbe_undeclared_parameters on some non-existent column.